src/views/Settings.vue

```diff
@@ -8,26 +8,27 @@
     </ion-header>
     
     <ion-content :fullscreen="true">
-      <!-- Profile of user logged in -->
+      <!-- Select store -->
       <ion-item>
-        <ion-icon :icon="personCircleOutline" slot="start" />
-        <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-        <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
+        <ion-icon :icon="storefrontOutline" slot="start" />
+        <ion-label>{{ currentFacility.facilityId ? currentFacility.facilityId : '' }}</ion-label>
```

Comment:
We should use value instead
https://github.com/hotwax/bopis/blob/main/src/views/Settings.vue#L15

Reply:
Updated the code.